phpstan.neon.dist

```diff
@@ -1,13 +1,14 @@
-parameters:
-    paths:
-        - %currentWorkingDirectory%/
-    excludes_analyse:
-          - %currentWorkingDirectory%/vendor/php-stubs/wordpress-stubs/wordpress-stubs.php 
-    autoload_files:
-#        # Procedural code
-         - %currentWorkingDirectory%/util.php
-         #- %currentWorkingDirectory%/altapay.php
-         #stubs
-         - %currentWorkingDirectory%/stubs/wc_get_product.php
-    autoload_directories:
-         - %currentWorkingDirectory%/
+parameters:
+    paths:
+        - %currentWorkingDirectory%/
+    excludes_analyse:
+        - %currentWorkingDirectory%/vendor/
+        - %currentWorkingDirectory%/assets/
+        - %currentWorkingDirectory%/tests/
+    scanDirectories:
+        - ../woocommerce/
+        - ../../../wp-includes/
+        - ../../../wp-admin/
+        - ../woocommerce-subscriptions
+    level: 0
+    checkMissingIterableValueType: false

```

Comment:
This won't have any effect at level 0 and will make the code less useful at level 6 (remove)

Reply:
removed checkMissingIterableValueType flag